src/Microsoft.ML.Transforms/Text/doc.xml

```diff
@@ -196,6 +196,12 @@
           using the output_tokens=True for TextTransform to convert a column with sentences like &apos;This is good&apos; into &apos;this&apos;, &apos;is&apos;, &apos;good&apos;.
           The suffix of &apos;_TransformedText&apos; is added to the original column name to create the output token column. For instance if the input column is &apos;body&apos;,
           the output tokens column is named &apos;body_TransformedText&apos;.</para>
+        <para>
+          WordEmbedding produce output column of floats with size of 3 * dimensionality of model. For example if you use GloVe50D we will produce 150 features.
+          First set of features would contain minumum values of encountered embeddings, second set will contain average values of encountered embeddings
+          and third set of features would contain maximum values of encounterd embeddings. The min/max provide a bounding hyper-rectangle for the words in the word embedding space.
+          This can assist for longer phrases where the average of many words drowns out the useful signal and your label is reasonable correlated with a dimension of the embedding space.
```

Comment:
```suggestion
          This can assist for longer phrases where the average of many words drowns out the useful signal and your label is reasonably correlated with a dimension of the embedding space.
```